include/El/core/imports/mpi/error.hpp

```diff
@@ -111,7 +111,7 @@ inline void MpiError(
     int mpi_error_code, std::string const& function,
     std::string const& file, int line)
 {
-    //El::break_on_me();
+    El::break_on_me();
```

Comment:
Again, seems like this should be commented out.

Reply:
It's supposed to be there.